core/src/main/java/org/apache/accumulo/core/metadata/schema/MetadataScanner.java

```diff
@@ -310,6 +312,18 @@ public RangeOptions scanTable(String tableName) {
       this.table = tableName;
       return this;
     }
+
+    @Override
+    public RangeOptions scanMetadataTable() {
+      this.table = MetadataTable.NAME;
+      return this;
```

Comment:
I always wonder, in these cases where the implementation for setting a single parameter is trivial, is it better to do what is here, or is it better to do:

```java
    @Override
    public RangeOptions scanMetadataTable() {
      return scanTable(MetadataTable.NAME);
    }
```

Reply:
I like that, its shorter.   I updated the code.